frontend/src/store/index.js

```diff
@@ -309,6 +309,20 @@ export const store = new Vuex.Store({
       return false
     },
 
+    is_liaison_designate(state) {
+      if(state.user.liaison_designate){
+        return true
+      }
+      return false
+    },
+
+    is_ita_designate(state) {
```

Comment:
Same goes for this one being defined up a bit.

Reply:
see previous comment